jpype/_jstring.py

```diff
@@ -40,16 +40,6 @@ class _JStringProto(object):
     def __add__(self, other):
         return self.concat(other)
 
-    def __eq__(self, other):
-        if isinstance(other, JString):
```

Comment:
These all fall back to comparable.  There may be interactions with hashcode.   This needed to clear CI on all versions.